### PR TITLE
Fix doc importer nits, re-run it.

### PR DESCRIPTION
**Pull request by thockin**
@erictune

### CODE REVIEW

**Comment by erictune** [comment]
lgtm

File: v1.0/docs/api-reference/definitions.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
----
```
Comment (nikhiljindal) [inline v1.0/docs/api-reference/definitions.md]: I just commented about this on https://github.com/GoogleCloudPlatform/kubernetes/pull/11717.
This change essentially reverts that PR.

Comment (nikhiljindal) [inline v1.0/docs/api-reference/definitions.md]: Also did you manually delete docs/api-reference/*.html?
The tool should have produced those files.

File: _tools/import_docs.sh [hunk 2]
```diff
@@ -30,9 +30,10 @@ for dir in docs examples; do
     find ${OUTDIR}/${dir} -type f -name \*.md | while read X; do
         sed -i \
             -e '/<!-- BEGIN STRIP_FOR_RELEASE.*/,/<!-- END STRIP_FOR_RELEASE.*/d' \
-            -e "s|releases.k8s.io/HEAD|releases.k8s.io/${REPORT_TAG}|" \
```
Comment (nikhiljindal) [inline _tools/import_docs.sh]: So is https://github.com/GoogleCloudPlatform/kubernetes/blob/master/build/versionize-docs.sh#L57 wrong as well?

Comment (thockin) [inline _tools/import_docs.sh]: Yes.  Can you please fix?

File: v1.0/docs/api-reference/definitions.md (hunk 1 above)
Comment (thockin) [inline v1.0/docs/api-reference/definitions.md]: I show those files.  I will fix the mess right now.  In general the  v1.0/docs and v1.0/exampes directories are slaved to the release-1.0 tree.  you MUST NOT add files in gh-pages that are not in release-1.0

**Pull request merged**